test/clj/rems/api/test_audit_log.clj

```diff
@@ -100,7 +105,7 @@
                        read-ok-body)))))
     (testing "filtering log by user"
       (is (= [{:userid "alice" :apikey "42" :method "get" :path "/api/users/active" :status "403"}
-              {:userid "alice" :apikey "42" :method "get" :path (str "/api/applications/" app-id) :status "200"}
+              {:apikey "42", :method "get", :path (str "/api/applications/" app-id), :status "200", :userid "alice"}
```

Comment:
the change on this line doesn't really do anythign does it? merely breaks the symmetry with the other lines

Reply:
Fixed